build_tester/tester.py

```diff
@@ -83,6 +84,21 @@ def __download_scripts(self):
         with open(os.path.join(self.config.install_dir_path, 'default.sh'), mode='r') as fs:
             default_script = fs.read()
 
+        if self.config.host_mode:
+            os_name = 'macos' if self.config.dist == 'os-x' else self.config.dist
```

Comment:
Why not just `osx`?

Reply:
Because on the Website API (where we get the installation commands) we use `os-x`